client/doc/mansrc/equo-mark.1.txt

```diff
@@ -0,0 +1,54 @@
+equo-mark(1)
+============
+:man source:   equo {equoversion}
+:man manual:   equo {equoversion}
+
+
+NAME
+----
+equo-mark - set properties on installed packages
+
+
+SYNOPSIS
+--------
+equo mark [-h] {auto,manual} ...
+
+
+INTRODUCTION
+------------
+Set properties on installed packages
+
+
+
+OPTIONS
+-------
+"equo mark" supports the following options which alters its behaviour.
+
+
+OPTIONAL ARGUMENTS
+~~~~~~~~~~~~~~~~~~
+*--help*::
+    show this help message and exit
+
+ACTION
+~~~~~~
+*auto*::
+    mark package as installed to satisfy a dependency
+
+*manual*::
+    mark package as installed by user
+
+
+
+AUTHORS
+-------
+Fabio Erculiani (lxnay@sabayon.org)
```

Comment:
Well, i wonder why not adding yourself here? :P you are author of this command at end

Reply:
This file is auto-generated (client/doc/generate).

I can update `generate` to pull author from docstring if you think it'll be useful (especially for future contributions). Maybe in another PR when I'd also update docs? :)